ambuda/models/auth.py

```diff
@@ -81,6 +81,40 @@ def __repr__(self):
         return f"<Role({self.id}, {self.name!r})>"
 
 
+class UserStatusLog(AmbudaUserMixin, Base):
+    """Tracks changes to user statuses."""
+
+    __tablename__ = "user_status_log"
+
+    #: Primary key.
+    id = pk()
+
+    #: The user whose status was changed.
```

Comment:
state changes should also indicate the user who made the change (or perhaps `ambuda-bot` if there isn't one)